selinux/Makefile.am

```diff
@@ -0,0 +1,32 @@
+SELINUX_POLICY_FILES = \
+	selinux/cockpit.fc \
+	selinux/cockpit.if \
+	selinux/cockpit.te \
+	$(NULL)
+
+SELINUX_POLICY_MANPAGES = \
+	selinux/cockpit_session_selinux.8cockpit \
+	selinux/cockpit_ws_selinux.8cockpit \
+	$(NULL)
+
+EXTRA_DIST += \
+	$(SELINUX_POLICY_FILES) \
+	$(SELINUX_POLICY_MANPAGES) \
+	$(NULL)
+
+cockpit.pp: $(SELINUX_POLICY_FILES)
+	$(AM_V_GEN) make -sf /usr/share/selinux/devel/Makefile cockpit.pp
```

Comment:
`$(MAKE)` for consistency?

Reply:
I'm not 100% sure about this.  This isn't building some other part of our package.  This is invoking a whole other build system of a separate thing.  I wanted to keep it isolated from anything that we might have configured.

On the other hand, I can't imagine it would hurt anything...

Reply:
I thought this was about running a different make; like calling `/opt/foo/make`, and expecting the same make to be used for everything. But anyway, bikeshedding. I'm fine with both.